spec/extensions/hints/result_spec.rb

```diff
@@ -52,6 +53,31 @@
       it "returns hints + error messages" do
         expect(result.messages.to_h).to eql(name: ["must be filled", "length must be within 2 - 4"])
       end
+
+      describe 'yaml translations backend' do
+        let (:yaml_messages) { <<~HEREDOC
+                                  en:
+                                    dry_validation:
+                                      errors:
+                                        filled?: "must be filled"
+                                        size?: "length must be within 2 - 4"
+
+                                  de:
+                                    dry_validation:
+                                      errors:
+                                        filled?: "de - must be filled"
+                                        size?: "de - length must be within 2 - 4"
+                                  HEREDOC
+                                }
```

Comment:
Could you reformat this so that it's a do...end block and the yaml uses less indentation?

Reply:
Yup i am on it